src/app/loadout/loadout-utils.ts

```diff
@@ -63,6 +67,27 @@ export function getLight(store: DimStore, items: DimItem[]): number {
   }
 }
 
+/** Returns a map of armor hashes to stats. There should be just one of each item */
+export function getArmorStats(
+  defs: D1ManifestDefinitions | D2ManifestDefinitions,
+  items: DimItem[]
+): { [hashHash: number]: DimCharacterStat } {
+  const statsByArmorHash = armorStats
+    .map((statHash) => defs.Stat.get(statHash))
+    .reduce((ret, { hash, displayProperties: { description, icon, name } }) => {
+      ret[hash] = { hash, description, icon: bungieNetPath(icon), name, value: 0 };
+      return ret;
+    }, {});
+
+  return items.reduce((ret, item) => {
+    const itemStats = _.groupBy(item.stats, (stat) => stat.statHash);
+    Object.keys(ret).forEach((hash) => {
+      ret[hash].value += itemStats[hash]?.[0].value ?? 0;
```

Comment:
Question regarding the stats shown, would we want to show the value or the baseValue here? I am pretty sure this will show the stats including equipped mods for the loadout. I don't have a preference, but thought it would be worth bringing up.

Reply:
ah good call, I think it should show what it actually would be when you apply it. basically what your new stats would update to in-game